virtaal/views/widgets/textbox.py

```diff
@@ -329,15 +331,16 @@ def move_elem_selection(self, offset):
     def __color_selector_textboxes(self, *args):
         """Put a highlighting border around the current selector text box."""
         if not hasattr(self, 'selector_color'):
-            self.selector_color = gtk.gdk.color_parse(current_theme['selector_textbox'])
+            self.selector_color = Gdk.color_parse(current_theme['selector_textbox'])
         if not hasattr(self, 'nonselector_color'):
-            self.nonselector_color = self.parent.style.bg[gtk.STATE_NORMAL]
+            self.nonselector_color = self.get_parent().get_style_context().get_color(Gtk.StateType.NORMAL)
 
         for selector in self.selector_textboxes:
             if selector is self.selector_textbox:
-                selector.parent.modify_bg(gtk.STATE_NORMAL, self.selector_color)
+                selector.get_parent().override_color(Gtk.StateType.NORMAL, Gdk.RGBA(*self.selector_color.to_floats()))
```

Comment:
This also looks like a change from background colour to foreground colour. Is this correct?